syfertext/span.py

```diff
@@ -260,4 +263,14 @@ def as_doc(self, owner: BaseWorker = None):  # tags: List[str] = None, descripti
             # Add token meta object to the new doc
             doc.container.append(self.doc.container[idx])
 
+        # Register the Doc on it's owners object store
+        doc.owner.register_obj(obj=doc)
```

Comment:
Maybe move this inside that `if` statement below it, because if we are working locally, we dont actually need to register to the `me` local worker

Reply:
Done.